pressbooks/pressbooks-book
# fix: change default color for block-toggle__cta_button to make visible

Fix for https://github.com/pressbooks/pressbooks-book/issues/913

To test:
1. Apply this branch and open a book homepage in a mobile view
2. Observe that the expand/show more arrows are now rendered in the network's primary color (rather than the primary _foreground_ color, which is often white!)
![Screenshot from 2021-12-16 18-16-06](https://user-images.githubusercontent.com/13485451/146477876-ae5442bd-a4a0-4218-adfc-c52103c306c5.png)
